optuna/artifacts/_list_artifact_meta.py

```diff
@@ -25,6 +25,45 @@ def get_all_artifact_meta(
             A storage object. This argument is required only if ``study_or_trial`` is
             :class:`~optuna.trial.FrozenTrial`.
 
+    Example:
+        An example where this function is useful:
+
+        .. code::
```

Comment:
This example seems to be unexecutable. It would be great to make it executable.

Reply:
As artifact creation tends to be a long code, I would like to avoid this here.
Plus, other APIs also use non-executable code blocks.